tensorflow/compiler/mlir/tosa/transforms/legalize_common.cc

```diff
@@ -3040,7 +3038,7 @@ std::optional<Value> convertReduceOpCommon(
     bool is_quantized, int32_t input_scale_multiplier,
     int32_t input_scale_shift, int64_t input_zp,
     int32_t output_scale_multiplier, int32_t output_scale_shift,
-    int64_t output_zp, bool keep_dims, StringRef nan_mode = "") {
```

Comment:
@sjarus @jpienaar ptal here. Not convinced this is right, and not familiar with `rewriter.getStringAttr("")`